llm4pddl/envs/assets/pddl/dressed/generate_tasks.py

```diff
@@ -0,0 +1,199 @@
+        'objs': suitjacket
+    }]
+    object_str = ''
+    for obj in objects:
+        if len(obj['objs']) > 0:
+            object_str += ' '.join(
+                obj['objs']) + f' - {obj["type"]}\n            '
+    # making initial predicates:
+    init_preds = ''
+    for person in persons:
+        init_preds += (f'(wearing-nothing-formal {person})\n         ' +
+                       f'(wearing-nothing-casual {person})\n         ')
+    for clothing in (dresses + sweatpants + sweatshirts + nicepants +
+                     collaredshirt + suitjacket):
+        init_preds += f'(in-closet {clothing})' + '\n         '
+    # making goal predicates:
+    goals = ''
+    goal_predicates = []
+    rng.shuffle(persons)
+    for i in range(num_casual_events):
+        goal_predicates.append(
+            f'(attending-casual-event {persons[i]})\n              ')
+    for i in range(
+            num_casual_events,
+        (num_casual_events + num_formal_in_dress + num_formal_in_suit)):
+        goal_predicates.append(
+            f'(attending-formal-event {persons[i]})\n              ')
+    rng.shuffle(goal_predicates)
+    for goal in goal_predicates:
+        goals += goal
+    #making problem:
+    problem_string = f"""(define (problem dressed)
+  (:domain dressed)
+  (:objects {object_str})
+  (:init {init_preds})
+  (:goal (and {goals}))
+  )"""
+    return problem_string
+
+
+def test_generate_dressing_problems() -> None:
+    """tests the function _generate_dressing_problems."""
+    rng = np.random.default_rng(seed=0)
+    prob1 = _generate_dressing_problems(2, 3, 2, 3, 0, 1, 0, 1, 0, 1, 1,
+                                        rng)[0]
+    rng2 = np.random.default_rng(seed=0)
+    reference_prob = _generate_dressing_problem(2, 2, 0, 0, 0, rng2)
+    assert prob1 == reference_prob
+
+
+def test_generate_dressing_problem() -> None:
+    """tests the function _generate_dressing_problem."""
+    rng = np.random.default_rng(seed=0)
+    out = _generate_dressing_problem(2, 2, 0, 0, 0, rng)
+    assert out == """(define (problem dressed)
+  (:domain dressed)
+  (:objects person1 person2 - person
+            sweatpants1 sweatpants2 - sweatpants
+            sweatshirt1 sweatshirt2 - sweatshirt
+            )
+  (:init (wearing-nothing-formal person1)
+         (wearing-nothing-casual person1)
+         (wearing-nothing-formal person2)
+         (wearing-nothing-casual person2)
+         (in-closet sweatpants1)
+         (in-closet sweatpants2)
+         (in-closet sweatshirt1)
+         (in-closet sweatshirt2)
+         )
+  (:goal (and (attending-casual-event person1)
+              (attending-casual-event person2)
+              ))
+  )"""
+
+
+if __name__ == "__main__":
+    import os
+    loc = os.path.dirname(os.path.realpath(__file__))
+    test_generate_dressing_problem()
+    test_generate_dressing_problems()
+
+    # 4 levels of difficulty:
+    rng_default = np.random.default_rng(seed=0)
+    thirty: List[str] = []
```

Comment:
minor: let's just call this variable `problems` or something like that, in case we ever want to change the number